comments.php

```diff
@@ -29,7 +29,7 @@
 	?>
 		<h3 class="comments-title">
 			<?php
-				printf( // WPCS: XSS OK.
+				printf( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Thanks for submitting this @bradp - can you keep the comments with your new ignores? ala:
```suggestion
				printf( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- XSS OK.
```